docs/managing_translations_i18n.md

```diff
@@ -0,0 +1,22 @@
+# Managing translations (i18n)
+
+## The workflow
+
+Decidim uses [Crowdin](https://crowdin.com/) to manage the translations.
+
+- Whenever someone [adds a new translation key](https://github.com/decidim/decidim/pull/1814/files#diff-c78c80097da59920d55b3f462ca21afaR177) to _Decidim_, _Crowdin_ gets notified and the new content is available to be translated from [Crowdin's Decidim dashboard](https://crowdin.com/project/decidim).
+- When a translator translates any key from Crowdin, it automatically creates a [PR in Github](https://github.com/decidim/decidim/pull/2207), adding the keys in the corresponding _yaml_ files.
```

Comment:
Instead of linking to a given PR, should we link to the list of Crowdin PRs?

https://github.com/decidim/decidim/pulls?utf8=%E2%9C%93&q=is%3Apr%20author%3Adecidim-bot%20Crowdin

Reply:
willdo!